### PR TITLE
Specify quantity unit via parser option.

### PR DESCRIPTION
**Pull request by brightbyte**
Bug: T77981

### CODE REVIEW

**Comment by tobijat** [comment]
@JanZerebecki @JeroenDeDauw @filbertkm @adrianheine

**Comment by JeroenDeDauw** [comment]
+1 (not +2)

**Comment by tobijat** [comment]
@thiemowmde

File: tests/ValueParsers/QuantityParserTest.php [hunk 1]
```diff
@@ -199,4 +200,24 @@ function( $number ) use ( $charmap ) {
...
+
+		$parser = new QuantityParser( $options );
+
+		$quantity = $parser->parse( '17' );
+		$this->assertEquals( 'kittens', $quantity->getUnit() );
+	}
+
+	public function testUnitOption_conflict() {
+		$options = $this->newParserOptions( array( QuantityParser::OPT_UNIT => 'kittens' ) );
+		$options->setOption( QuantityParser::OPT_UNIT, 'kittens' );
```
Comment (thiemowmde) [inline tests/ValueParsers/QuantityParserTest.php]: Duplicate, fixed in #23.

**Pull request merged**